srcpkgs/libGL/patches/fix-i686-cross.patch

```diff
@@ -0,0 +1,11 @@
+--- src/mesa/meson.build
++++ src/mesa/meson.build
+@@ -607,6 +607,7 @@
+     'x86/gen_matypes.c',
+     c_args : [c_vis_args, c_msvc_compat_args],
+     include_directories : inc_common,
++	native : true,
```

Comment:
Although not important in a patch, I'm slightly triggered because the indention is messed up.